spec/03-plugins/06-statsd/01-log_spec.lua

```diff
@@ -655,6 +655,31 @@ for _, strategy in helpers.each_strategy() do
     end)
 
     describe("metrics", function()
+      -- the purpose of this test case is to test the batch queue
+      -- finishing processing its batch in one time (no retries)
+      it("no more metrics than expected", function()
+        ---[[ EE
+        local metrics_count = 14
+        --]] EE
```

Comment:
What's the purpose of these codes? We can add the EE-specific assertion when cherry-picking this to EE.

Reply:
sorry, mistake

Reply:
Let's create a cherry-pick PR to EE when this gets merged.